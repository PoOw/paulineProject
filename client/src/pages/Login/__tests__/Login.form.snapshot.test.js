// @flow
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import InnerLoginForm from '../Login.form';

describe('The form component', () => {
  it('should render correctly', () => {
    const props = { errors: {}, touched: {}, isSubmitting: false };
    const tree = shallow(<InnerLoginForm {...props} />);
    expect(toJSON(tree)).toMatchSnapshot();
  });
});
