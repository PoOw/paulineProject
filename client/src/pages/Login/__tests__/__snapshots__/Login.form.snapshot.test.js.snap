// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The form component should render correctly 1`] = `
<Form>
  <div>
    <C
      name="email"
      type="email"
    />
  </div>
  <div>
    <C
      name="password"
      type="password"
    />
  </div>
  <button
    disabled={false}
    type="submit"
  >
    Connect
  </button>
</Form>
`;
